### PR TITLE
chore(react): migrate deprecated React.PropTypes

### PR DESCRIPTION
**Pull request by nLight**
Enabling react@16

PropTypes have been extracted in a separate package `prop-types`. It still works with both, can be merged before changing the plugins.

### CODE REVIEW

**Comment by nLight** [comment]
Alfred test this

**Comment by nLight** [comment]
Alfred test this

**Comment by juliangieseke** [comment]
Adding WIP label because its not touched for more then a week and has conflicts.

**Comment by bstavroulakis** [comment]
@nLight @juliangieseke  I rebased, fixed conflicts and forced pushed. Let us see what the tests will tell us now.

**Comment by bstavroulakis** [comment]
@juliangieseke Had to rebase again 😅 . Let us get this merged today because it will always create conflicts.

**Comment by bstavroulakis** [comment]
Alfred please test this

**Comment by DanielMSchmidt** [comment]
I feel like the changeset is so big that we should take the time to resolve the conflicts and immediately review it afterwards to get it merged. Please ping me once the rebase is done and I will do the review 👍

**Comment by juliangieseke** [comment]
yes, we will tackle this one when the test-refactoring PR is merged 👍

**Comment by nLight** [comment]
rebased 😏

**Comment by nLight** [comment]
Alfred test this

File: src/js/routes/factories/network.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 /* eslint-disable no-unused-vars */
-import React, { PropTypes } from "react";
+import PropTypes from "prop-types";
```
Comment (DanielMSchmidt) [inline src/js/routes/factories/network.js]: I think we shouldn't have PropTypes inside the eslint-disable, but that is no blocker for me, we can adjust afterwards 👍

**Comment by juliangieseke** [comment]
Alfred test this

**Comment by juliangieseke** [comment]
Alfred test this

**Comment by juliangieseke** [comment]
Alfred test this

**Comment by juliangieseke** [comment]
Alfred test this

**Pull request merged**